### PR TITLE
DCOS-22349: add gpu in nodes table

### PR DESCRIPTION
**Pull request by bstavroulakis**
Add GPU in nodes table.

Closes DCOS-22349

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by DanielMSchmidt** [comment]
@juliangieseke someone did 🤷‍♂️

**Comment by bstavroulakis** [comment]
@juliangieseke @DanielMSchmidt I added the ticks. Sorry about that.

File: plugins/nodes/src/js/components/NodesTable.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ import UnitHealthUtil from "#SRC/js/utils/UnitHealthUtil";
 const COLOR_CLASSNAMES = {
   cpus: "color-1",
   mem: "color-2",
-  disk: "color-3"
+  disk: "color-3",
+  gpus: "color-1"
 };
```
Comment (leemunroe) [inline plugins/nodes/src/js/components/NodesTable.js]: @bstavroulakis I believe the intent here was to align the colors so each resource is the same color and they would be green/yellow/red depending on the percentage. Looking at this code I'm guessing that is not the case.

cc @lilysamimi

**Comment by bstavroulakis** [comment]
@ahoskins I follow the same pattern as we already had. Made no assumptions for style changes

image

File: plugins/nodes/src/js/components/NodesTable.js (hunk 1 above)
Comment (bstavroulakis) [reply plugins/nodes/src/js/components/NodesTable.js]: Hm, this seems not to be the case, the code assigns each resource with a fixed color value

**Comment by lilysamimi** [comment]
Head up @bstavroulakis I'm waiting on further clarification from @nLight and @orlandohohmeier on this.

**Comment by DanielMSchmidt** [comment]
I think the spacing is not right yet, but let's fix that forward maybe ^^ 

gpu

**Comment by DanielMSchmidt** [comment]
I think the spacing is not right yet, but let's fix that forward maybe ^^

**Comment by DanielMSchmidt** [comment]
Added hold merge due to @lilysamimi comment

**Pull request merged**